src/components/Header/Header.js

```diff
@@ -21,13 +21,13 @@ const headerLinks = [
 
 function Header() {
   return (
-    <HeaderWrapper>
+    <HeaderWrapper data-testid="header">
       <Link to="/">
-        <Logo src={LogoSVG} alt="Logo" />
+        <Logo src={LogoSVG} alt="Logo" data-testid="header-logo" />
       </Link>
-      <nav>
-        {headerLinks.map((item) => (
-          <NavLink key={item.title} to={item.link} alt={item.title}>
+      <nav data-testid="navbar">
```

Comment:
I've been thinking of doing that, but I'm sitting with a mental dilemma.

I would have to test for the strings in the links in the header, but then these tests will fail if someone ever changes the contents of the links. I thought that tests shouldn't be written this way, something along the lines of "tests shouldn't break if someone changes the content of the site".

Reply:
I haven't been able to make this change. I will handle the others and come back to it.

Reply:
That's a good point. At the same time the wordings don't change so often typically. And if somebody changed the text inside an important link or button the tests would catch it.

Honestly, I think there are advantages to both approaches. I'd just go for the recommended one because of the reasons above and because you don't need so many test IDs in your components